NWPRTXPS/NWPRTXPS-IOC-01App/src/build.mak

```diff
@@ -72,11 +72,6 @@ $(APPNAME)_SRCS_vxWorks += -nil-
 #$(APPNAME)_OBJS_vxWorks += $(EPICS_BASE_BIN)/vxComLibrary
 
 # Finally link to the EPICS Base libraries
-## area detector already includes PVA, so avoid including it twice
-ifeq ($(AREA_DETECTOR),)
-include $(CONFIG)/CONFIG_PVA_ISIS
-endif
-
 $(APPNAME)_LIBS += $(EPICS_BASE_IOC_LIBS)
```

Comment:
This change should be discarded - it is because the instrument had an older version of the file